src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlType.java

```diff
@@ -180,4 +182,31 @@ void generatePrecision() {
         int maxPrecision = this.precision;
         this.precision = ThreadLocalRandom.current().nextInt(minPrecision, maxPrecision + 1);
     }
+
+    /**
+     * generate char or nchar values
+     * 
+     * @param columnLength
+     * @param charSet
+     * @return
+     */
+    protected static String buildCharOrNChar(int columnLength, String charSet) {
```

Comment:
move these new methods into SqlChar class, as this is an abstract class and we are not going to use these methods in all the classes that extends SqlType.